README.md

```diff
@@ -46,32 +58,44 @@ The bridge is also shaped for liquidity providers who are able to assume Ethereu
-  *  `Proxy` - A proxy implementation 
+  *  `Bridge` -  handles rewards update, deposit & withdrawal of
+     `static_a_tokens`, their corresponding `aTokens` and their underlying
+     assets
+  *  `Proxy` - A proxy implementation
 
 `L2`
-  * `staticAToken` - token wrapper converting balance-increasing `aTokens` into exchange-rate-increasing on Starknet.
-  * `claimable` - tracks users' claimable rewards and current reward index for each `ETHStaticAToken`
-  * `rewAAVE` - an ERC20 representing the rewards on L2
-  * `token_bridge` - is responsible for:
-    * minting and burning staticATokens on message from L1. 
-    * bridging rewAAVE tokens back to L1
-    * updating `rewards_index` for each staticAToken on message from L1 
-  *  `proxy` - a generic implementation of a proxy in cairo
-
-## StaticATokens on L2
-
-Natively, Aave tokens grow in balance, not in value. To be able to create this kind of model, it is important to wrap them before bridging, converting them in a token that grows in value, not in balance.
-
-staticATokens are an implementation of the wrapped aTokens that will continuously increase in value on Starknet because they are backed by the increasing aTokens amounts locked in the bridge contract on Ethereum. staticATokens can then be bridged back to aTokens.
+  * `static_a_token` - exchange-rate-increasing wrapper of `aTokens` on
+    Starknet
+  * `incentivized_erc20` - tracks users' claimable rewards and current reward index for
+    each `static_a_token`
+  * `rewAAVE` - ERC20 representing the rewards on L2
+  * `bridge` - bridge responsible for:
+    * minting and burning `static_a_tokens` on message from L1
+    * bridging `rewAAVE` tokens back to L1
+    * updating `rewards_index` for each `static_a_token` on message from L1
+  *  `proxy` - generic implementation of a proxy in cairo
+
+## static_a_tokens on L2
+
+Natively, AAVE tokens grow in balance, not in value. To be able to create this
+kind of model, it is important to wrap them before bridging, converting them in
+a token that grows in value, not in balance.
+
+`static_a_tokens` are an implementation of the wrapped `aTokens` that will
+continuously increase in value on Starknet because they are backed by the
+increasing `aTokens` amounts locked in the bridge contract on Ethereum.
+`static_a_tokens` can then be bridged back to `aTokens`.
 
 ## Bridging aTokens from L1<>L2 <a name="bridging-atokens-from-l1--l2"></a>
 
 ### Transfer L1->L2: <a name="transfer-l1--l2-"></a>
 
 
-Users can either bridge their aToken (let's say aDai) to L2 by calling `deposit()` on `TokenBridge`, or deposit the underlying asset (i.e Dai).
-Users wil have to approve the bridge to spend the underlying `asset` tokens or `aTokens`, depending on the provided value for `fromAsset` argument when depositing.
+Users can either bridge their `aToken` (let's say aDai) to L2 by calling
+`deposit()` on `Bridge`, or deposit the underlying asset (i.e Dai). Users
```

Comment:
Users can either bridge their `aToken` (let's say aDai) to L2 or deposit the underlying asset (i.e. Dai) by calling `deposit` on `Bridge`.

Reply:
Two comments: 
- both behavior are possible on the function `deposit` once [this PR](https://github.com/aave-starknet-project/aave-starknet-bridge/pull/72) is merged,
- the function name is `deposit` and not `deposit()`. and `deposit` function needs 5 arguments, so writing `deposit()` is misleading.

Reply:
I don't think mentioning all 5 arguments is relevant to the user, what should be highlighted here is the possibility to deposit from underlying asset

Reply:
sure, I totally agree with you! I just meant that code-wise, `deposit()` does not exist, so I think `deposit` or `deposit(...)` is a better name. anyway, this is a detail :)